tests/Spec.hs

```diff
@@ -696,7 +699,8 @@ main = hspec $ do
               Project.author              = "rachael",
               Project.approvedBy          = Just "deckard",
               Project.integrationStatus   = Project.NotIntegrated,
-              Project.integrationAttempts = []
+              Project.integrationAttempts = [],
+              Project.needsFeedback       = False
             }
           prMap = IntMap.fromList [(1, pullRequest1), (2, pullRequest2)]
           -- After a successful push, the state of pull request 1 will still be
```

Comment:
We should also add tests for the cases mentioned in https://github.com/channable/hoff/issues/53#issuecomment-659278435. It’s a good idea to add a failing test first, to make sure that we can reproduce the bug. Then after implementing the fix, the test should pass.

Reply:
In fd95f6c I added a test that reproduces the bug (I think), could you look at that?

It fails with the following message:

```
  1) Logic.proceedUntilFixedPoint reports the build status if a user retries the same commit
       expected: [AIsReviewer (Username "deckard"),ALeaveComment (PullRequestId 1) "Pull request approved by @deckard, rebasing now.",ATryIntegrate "Merge #1\n\nApproved-by: deckard" (refs/pull/1/head,a39),ALeaveComment (PullRequestId 1) "Rebased as b71, waiting for CI \8230",ALeaveComment (PullRequestId 1) "The build failed.",AIsReviewer (Username "deckard"),ALeaveComment (PullRequestId 1) "The build failed."]
        but got: [AIsReviewer (Username "deckard"),ALeaveComment (PullRequestId 1) "Pull request approved by @deckard, rebasing now.",ATryIntegrate "Merge #1\n\nApproved-by: deckard" (refs/pull/1/head,a39),ALeaveComment (PullRequestId 1) "Rebased as b71, waiting for CI \8230",ALeaveComment (PullRequestId 1) "The build failed.",AIsReviewer (Username "deckard"),ALeaveComment (PullRequestId 1) "Pull request approved by @deckard, rebasing now."]
```

Reply:
Yes, that test looks good!